app/src/main/java/com/amaze/filemanager/ui/fragments/data/MainFragmentViewModel.kt

```diff
@@ -141,15 +143,9 @@ class MainFragmentViewModel : ViewModel() {
      * Initialize column number from preference
      */
     fun initColumns(sharedPreferences: SharedPreferences) {
-        if (columns == 0) {
-            sharedPreferences.getString(
-                PreferencesConstants
-                    .PREFERENCE_GRID_COLUMNS,
-                "-1"
-            )?.toInt()?.run {
-                columns = this
-            }
-        }
+        val columnPreference = sharedPreferences.getString(PREFERENCE_GRID_COLUMNS, PREFERENCE_GRID_COLUMNS_DEFAULT)
+        Objects.requireNonNull(columnPreference)
```

Comment:
` Objects#requireNonNull` would throw an NPE if `columnPreference` is `null`. Did you mean `if(columnPreference != null)`?

Reply:
It will never be null.